core/embed/trezorhal/haptic.h

```diff
@@ -23,4 +23,11 @@ bool haptic_test(uint16_t duration_ms);
 // Play haptic effect
 void haptic_play(haptic_effect_t effect);
```

Comment:
Not sure about this: but what about renaming `haptic_play()` and `haptic_play_rtp()` to something that better explains their function - like `haptic_play_effect()` and `haptic_play()` instead of `haptic_play_rtp()`?

Reply:
renamed to `haptic_play_custom` in https://github.com/trezor/trezor-firmware/commit/8f4877998e03dd176e3768f1f3a84e8a8bcaec13

(its still not ideal as the values passed might need to be different if different driver IC or actuator is used, but i don't see a very clear way to deal with this now and there is no pressing need to have it solved right now)